bin/setup

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
+
+# Exit if any subcommand fails
+set -e
+
+STEPS=5
+
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[0;33m'
+NO_COLOR='\033[0m'
+CLEAR_LINE='\r\033[K'
+
+# STEP 1
+printf "[1/${STEPS}]  checking dependencies"
+
+actual_node_version=$(node --version)
+expected_node_version=$(cat .nvmrc)
+
+if [ "$actual_node_version" != "$expected_node_version" ]; then
+  printf "${CLEAR_LINE}${RED}  Your node version must match before setup can continue${NO_COLOR}\n"
+  printf "    You should:\n"
+  printf "      $ nvm install $expected_node_version\n"
```

Comment:
That’s a good point, but I think that in this case everybody (at Artsy) is using `nvm`. My main issue is that it then lines up with the example `nvm install …` command, otherwise it could be confusing to a newcomer if the suggested command doesn’t actually work.